### ISSUE TITLE
After setup round in the empty game Brooke only had 1 road and 1 settlement

### ISSUE THREAD

**Issue by dolson25** [cs340tabyu/cs340Winter2017#284]

**Comment by mhnsn** [comment]
Poor Brooke.

**Comment by dolson25** [comment]
Haha Mark.   Actually felt like I should clarify after testing some more.  In general a couple of the players in the middle of the setup rotation will have missing roads and/or settlements.  Not sure the exact error but it's easy to replicate a similar error.

**Comment by csydvs** [comment]
I can't replicate this issue--here are two different attempts. Perhaps there's some sort of a sporadic server error skipping turns on your OS:
![image](https://cloud.githubusercontent.com/assets/15003465/19791140/aa58e5e4-9c7b-11e6-832b-6c6c5868a5ef.png)
![image](https://cloud.githubusercontent.com/assets/15003465/19791228/68237f6c-9c7c-11e6-98f6-d772693c07ef.png)

**Comment by dolson25** [comment]
Maybe it's just on my OS.  I'm uploading this screenshot I just took so the TA's know I wasn't a crazy tester.  This is after opening a client, logging in as Sam,  and playing a settlement and a road.  The client then said I was waiting on Pete, so I opened a client for him and played.  Then I played for Mark twice, Pete again, and then Sam, including his first roll.  It's now Brooke's turn but she doesn't have anything yet

![catan 2](https://cloud.githubusercontent.com/assets/17937610/19792298/2e188722-9c86-11e6-947a-be228243902e.PNG)
